rpcd/playbooks/roles/rpc_setup_infrastructure/tasks/sshd.yml

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
Of course if this was just added to the support role, and support/deploy.sh just was able to override the target hosts for the playbook which implements the support role then you'd have a far simpler solution.